source/features/bypass-checks.tsx

```diff
@@ -15,12 +16,20 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 	}
 }
 
-async function init(): Promise<void> {
+function init(): void {
 	// This selector excludes URLs that are already external
-	const thirdPartyApps = select.all<HTMLAnchorElement>('a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="/"]');
+	const thirdPartyApps = [
+		`a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="${location.origin}"]:not(.rgh-bypass-link)`, // Hovercard status checks
+		'a:not([href="/apps/github-actions"]) ~ div .status-actions[href^="/"]:not(.rgh-bypass-link)'
+	].join();
 
-	// If anything errors, RGH will display the error next to the feature name
-	await Promise.all(thirdPartyApps.map(bypass));
+	observe(thirdPartyApps, {
+		constructor: HTMLAnchorElement,
+		add(thirdPartyApp) {
+			thirdPartyApp.classList.add('rgh-bypass-link');
+			void bypass(thirdPartyApp);
```

Comment:
I haven't written code in a while, but do we need `void` here? Just curious!

Reply:
It’s due to an ESLint rule to avoid calling async functions but forgetting to await them. `void` is a way to specify that we don’t want to await it.